backend_server/Dockerfile

```diff
@@ -6,6 +6,10 @@ COPY . /python-back-end
 
 RUN pip install -r requirements.txt
 
+RUN adduser --disabled-password --gecos '' backendserver
+RUN chown -R backendserver:backendserver /python-back-end
+USER backendserver
+
 CMD python server.py --database $DATABASE --host $HOST --user $USER --pw $PASSWORD  --port $PORT
 
-EXPOSE 5000
\ No newline at end of file
+EXPOSE 5000
```

Comment:
If the backend_url is configured with an env variable, should we also have a variable here to change the exposed port, if someone wants to expose something other than 5000? @ville-friman-siili

Reply:
Excellent point. I was so focused on the frontend part that forgot this. I'll change this before merging.